components/CardList.js

```diff
@@ -0,0 +1,18 @@
+import { medications } from "@/lib/db";
+import styled from "styled-components";
+import Card from "./Card";
+
+const List = styled.ul`
+  list-style: none;
+  padding: 0;
+`;
+
+export default function CardList() {
+  return (
+    <List>
+      {medications.map((medication) => (
+        <Card key={medication.id} medication={medication} />
```

Comment:
You removed the `li`, but this should not be removed. Please wrap your Card component in a `li`. 👀

Reply:
Thank you , for you. feedback 
I'll chaange it now